packages/airnode-deployer/terraform/gcp/main.tf

```diff
@@ -104,22 +106,22 @@ module "httpReq" {
   source = "./modules/function"
   count  = var.http_api_key == null ? 0 : 1
 
-  name               = "${local.name_prefix}-httpReq"
-  entry_point        = "httpReq"
-  source_dir         = var.handler_dir
-  memory_size        = 128
-  timeout            = 15
-  configuration_file = var.configuration_file
-  secrets_file       = var.secrets_file
-  region             = var.gcp_region
-  project            = var.gcp_project
-
+  name                  = "${local.name_prefix}-httpReq"
+  entry_point           = "httpReq"
+  source_dir            = var.handler_dir
+  memory_size           = 128
+  timeout               = 15
```

Comment:
I just noticed this, we give the handler 15 seconds for HTTP gateway requests but when run via the coordinator API calls can have 30 seconds? I can't remember why this (shouldn't they be the same)? #educate_me 😛

Reply:
> I can't remember why this (shouldn't they be the same)?

I have no clue :smile: I agree, they should be the same. Should I increase it?

> It looks like both GCP and AWS max-out at 30 seconds for HTTP gateway based requests.

That's strange if we have the limit set to 15 seconds, isn't it?

Reply:
> I have no clue 😄 I agree, they should be the same. Should I increase it?
I think so, it should probably be 30 seconds.

> That's strange if we have the limit set to 15 seconds, isn't it?
I don't understand :P There's a hard-limit on signed data gateway requests of 30 seconds imposed by the cloud providers - so it's possible for the handler to be timed out by AWS/GCP because our underlying `go` based timeouts are also 30s I think.